.travis.yml

```diff
@@ -1,34 +1,57 @@
+env:
+  - ARCH=amd64
+    DOCKERFILE=Dockerfile
+  - ARCH=arm32v7
+    DOCKERFILE=Dockerfile.armhf
+  - ARCH=arm64v8
+    DOCKERFILE=Dockerfile.arm64
+
 language: go
+
 required: sudo
+
 go:
-- '1.13'
+  - '1.13'
+
 services:
-- docker
-- ntp
-- xvfb
+  - docker
+  - ntp
+  - xvfb
+
 addons:
   chrome: stable
   apt:
     sources:
-    - google-chrome
+      - google-chrome
     packages:
-    - libgif-dev
-    - google-chrome-stable
+      - docker-ce
+      - libgif-dev
+      - google-chrome-stable
 
 before_script:
-- curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
-- nvm install v11 && nvm use v11 && npm i
+  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+  - nvm install v11 && nvm use v11 && npm i
+  - docker run --rm --privileged multiarch/qemu-user-static:register --reset -p yes
```

Comment:
Please comment what this command is supposed to do. I'm not quite sure by just reading it. Don't hesitate to also paste the link where you found it if it's the case.

Reply:
[multiarch/qemu-user-static](https://github.com/multiarch/qemu-user-static) is to enable an execution of different multi-architecture containers by QEMU and binfmt_misc.